integration_test/tests/mongosqltranslate_tests.rs

```diff
@@ -0,0 +1,171 @@
+mod common;
+
+mod mongosqltranslate {
+    use crate::common::{
+        allocate_env, connect_with_conn_string, default_setup_connect_and_alloc_stmt,
+        disconnect_and_close_handles, fetch_and_get_data, get_column_attributes,
+        get_sql_diagnostics,
+    };
+    use cstr::WideChar;
+    use definitions::{
+        AttrOdbcVersion, CDataType, HStmt, Handle, HandleType, SQLExecDirectW, SQLExecute,
+        SQLPrepareW, SqlReturn, SQL_NTS,
+    };
+
+    #[test]
+    fn test_srv_style_uri_connection() {
+        let env_handle = allocate_env(AttrOdbcVersion::SQL_OV_ODBC3);
+        let conn_str =
+            crate::common::generate_srv_style_connection_string(Some("test".to_string()));
+        let result = connect_with_conn_string(env_handle, Some(conn_str));
+
+        assert!(
+            result.is_ok(),
+            "Expected successful connection, got error: {:?}",
+            result
+        );
+
+        let _ = unsafe { Box::from_raw(env_handle) };
+    }
+
+    #[test]
+    fn test_sql_prepare_and_sql_execute_with_library_loaded_and_valid_query_and_valid_schemas_created(
+    ) {
+        let (env_handle, dbc, stmt) = default_setup_connect_and_alloc_stmt(
+            AttrOdbcVersion::SQL_OV_ODBC3,
+            Some(crate::common::generate_srv_style_connection_string(Some(
+                "sample_airbnb".to_string(),
+            ))),
+        );
+
+        unsafe {
+            let mut query: Vec<WideChar> = cstr::to_widechar_vec("SELECT property_type, room_type, bed_type, minimum_nights, maximum_nights FROM listingsAndReviews LIMIT 3");
+            query.push(0);
+
+            assert_eq!(
+                SqlReturn::SUCCESS,
+                SQLPrepareW(stmt as HStmt, query.as_ptr(), SQL_NTS),
+                "{}",
+                get_sql_diagnostics(HandleType::SQL_HANDLE_STMT, stmt as Handle)
+            );
+
+            get_column_attributes(stmt as Handle, 5);
```

Comment:
Since we're not running the result_set_tests on mongosqltranslate, we don't check the actual outputs only that the counts match.  Can you see if here we can add a check that we are getting the actual column values we expect?  

That would help confirm we're getting the correct values from the change made in getting the schema information.  Let me know if you think that would be helpful.  In the result set test we do it [here](https://github.com/mongodb/mongo-odbc-driver/blob/52be409a03589324e1401501925b1e3c86ced764/integration_test/src/test_runner/mod.rs#L580).